### PR TITLE
Add intuitive trash

### PR DESCRIPTION
**Pull request by iamunknown2**
Add a trash can that shows when block clicked and hides when block click is released.

The trash can in the bottom-right hand corner is still there.

### CODE REVIEW

File: resources/js/blockFilter.js [hunk 1]
```diff
@@ -93,13 +93,6 @@ var filter = {
 
     /* Blocks for palette 0 - text */
     {
-      // Moved text node to top because it is basically the most used block
```
Comment (matthewr6) [inline resources/js/blockFilter.js:96]: Why did you remove so many blocks?

File: index.html [hunk 2]
```diff
@@ -137,13 +137,13 @@
                 </label>
                 <label class="paletteOptionWrap" for="p4" data-num="3">
                   <input type="radio" name="palette" value="p4" id="p4" />
-                  tables<span class="paletteColorIndicator" style="background:#4D75E4"></span>
+                  Palette 4<span class="paletteColorIndicator" style="background:#4D75E4"></span>
```
Comment (matthewr6) [inline index.html:140]: Why did you switch back to the generic palette name?

**Comment by iamunknown2** [comment]
@Firedrake969 Uh I just replaced everything in gh-pages with the previous block-editor-intuitive-trash files/folders. I've tested it before with merging and the newer blocks still appear fine.

**Comment by matthewr6** [comment]
However, it seems like changes were reverted, and that's not OK...
I might be harsh but I don't want a minor change reverting other changes.
